### DISCUSSION TITLE
How to group agent

### DISCUSSION THREAD

**Discussion by Huang0123** [apache/skywalking#9998] [Q&A]
How can agents be grouped according to different items and displayed on the UI page? If yes, how can they be configured? Looking forward to your reply，Thank you

**Comment by wu-sheng** [comment]
`according to different items` What do you mean? Group is only a name prefix, https://skywalking.apache.org/docs/main/v9.2.0/en/setup/backend/service-auto-grouping/

**Reply by Huang0123** [reply]
Agent grouping means that different projects are deployed on the container, and microservices can be grouped according to different projects, such as different projects (first level directories), and microservices (second level directories). At last, they can be displayed on the UI page. At present, the names ofagent are not mixed together, making it inconvenient to view which microservices under this project deploy agent

**Reply by wu-sheng** [reply]
An open-source project is not for following your preference, it is going to resolve tech problems. 
If you want something for your private usage, you need to do that on your own. Especially, management requirements are super different from various vendors and orgs, so, don't expect us to follow those.
